src/Servers/Kestrel/test/InMemory.FunctionalTests/HttpsTests.cs

```diff
@@ -402,15 +402,15 @@ public async Task DevCertWithInvalidPrivateKeyProducesCustomWarning()
                 using (var sslStream = new SslStream(connection.Stream, true, (sender, certificate, chain, errors) => true))
                 {
                     // SslProtocols.Tls is TLS 1.0 which isn't supported by Kestrel by default.
-                    await Assert.ThrowsAsync<IOException>(() =>
+                    await Assert.ThrowsAnyAsync<Exception>(() =>
                         sslStream.AuthenticateAsClientAsync("127.0.0.1", clientCertificates: null,
                             enabledSslProtocols: SslProtocols.Tls,
                             checkCertificateRevocation: false));
                 }
             }
 
             await loggerProvider.FilterLogger.LogTcs.Task.DefaultTimeout();
-            Assert.Equal(3, loggerProvider.FilterLogger.LastEventId);
+            Assert.Equal(1, loggerProvider.FilterLogger.LastEventId);
```

Comment:
Not exactly sure why this needed to change. But it lines up w/ tests in later branches